portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello, Omar!</h1>");
```

Comment:
Want to add to the comment above that this returns a placeholder comment saying 'Hello, Omar!' while keeping the TODO?

Also worth updating the TODO so that it says TODO(oumontiel). It'll be good practice for when you are working on code with the rest of the pod to track which TODOs are for which people.

Reply:
Done! Didn't know about this practice, will keep it in mind.